README.md

```diff
@@ -2,7 +2,7 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/portworx/ark-plugin)](https://goreportcard.com/report/portworx/ark-plugin)
 
 
-# ark-plugin
-Portworx plugin for Heptio Ark
+# velero-plugin
+Portworx plugin for Heptio Velero (formerly known as Ark)
 
 Instructions to deploy and use can be found here: https://docs.portworx.com/scheduler/kubernetes/ark.html
```

Comment:
It would be awesome if you could update the repository name and documentation to use "velero" in place of "ark".

Reply:
Will move it as a separate change, need to rename the repo in github first.